numpy/__init__.pyi

```diff
@@ -439,7 +438,6 @@ from numpy.lib._function_base_impl import (
     angle as angle,
     unwrap as unwrap,
     sort_complex as sort_complex,
-    disp as disp,
```

Comment:
There is a deprecated remnant of this, I don't think there is a reason for it.  But since it is deprecated, it shouldn't have a typing stub either way.

Reply:
That makes sense; I'll remove the stubs for `numpy._function_base_impl.disp` 👌🏻